rules/autoconsent/povr.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "povr",
+    "vendorUrl": "https://povr.com/",
+    "cosmetic": false,
+    "runContext": {
+        "urlPattern": "^https://povr\\.com/"
+    },
+    "prehideSelectors": [],
+    "detectCmp": [
+        {
+            "exists": ".alert .accept-cookies,form.js-cookies"
+        }
+    ],
+    "detectPopup": [
+        {
+            "visible": ".alert .accept-cookies,form.js-cookies"
+        }
+    ],
+    "optIn": [
+        {
+            "waitForThenClick": ".alert .accept-cookies,form.js-cookies button[type=submit]"
+        }
+    ],
+    "optOut": [
+        {
+            "hide": ".alert:has(.accept-cookies)"
+        },
+        {
+            "if": { "exists": "form.js-cookies button" },
+            "then": [
+                {
+                    "waitFor": "form.js-cookies input[type=checkbox]"
+                },
+                {
+                    "click": "form.js-cookies input[type=checkbox]:checked:not(:disabled)",
+                    "all": true,
+                    "optional": true
+                },
+                {
+                    "waitForThenClick": "form.js-cookies button[type=submit]"
+                },
+                {
+                    "eval": "EVAL_POVR_GOBACK"
```

Comment:
Won't this break the lifecycle? Did one of the steps about trigger a navigation?

Reply:
yeah, the settings are on a separate page, clicking on the link from [line 46](https://github.com/duckduckgo/autoconsent/pull/659/files#diff-0b4ce63f4fa437b1ed74d542996e2310fd93d5d1425c539519e112ec8b7edd7fR46) navigates there.

Reply:
But won't the autoconsent lifecycle restart after navigation?

Reply:
no, because the popup on the first page would be hidden already, and it shouldn't trigger again